android/assets/jsons/translations/French.properties

```diff
@@ -739,13 +739,13 @@ Closest city = Ville la plus proche
 Action = Action
 Defeated = Vaincu
  # Requires translation!
-[numberOfCivs] Civilizations in the game = 
+[numberOfCivs] Civilizations in the game = [numberofCivs] Civilisation dans la partie 
```

Comment:
`[numberOfCivs] Civilizations in the game = [numberOfCivs] Civilisation dans la partie `